Kitodo/src/main/java/org/kitodo/production/helper/metadata/legacytypeimplementations/LegacyInnerPhysicalDocStructHelper.java

```diff
@@ -37,11 +37,15 @@
     private static final MetsService metsService = ServiceManager.getMetsService();
 
     /**
-     * A constant for local use. We cannot make this constant constant because
-     * the service loader is an instance variable.
+     * A constant for local use.
      */
     @Deprecated
-    public final MediaVariant local = new MediaVariant();
+    public static final MediaVariant LOCAL = new MediaVariant() {
+        {
+            setUse("LOCAL");
+            setMimeType("image/tiff");
```

Comment:
Should setting the mime type not depend of which kind of file is really used as local file? So far as I know institutes are using jpeg files for local usage and their tiff files are not used.

Reply:
This is only a temporary solution as long as the helper classes for UGH replacement still exist. The value has always been written that way. It has never been used and read, it just needs to be in the field so there is no NullPointerException. Everything is configurable in the final solution. You do not even have to call the group LOCAL anymore.